### PR TITLE
feat: multi languages support #266

### PR DESCRIPTION
**Pull request by ifaouibadi**

### CODE REVIEW

**Review by kenodressel — APPROVED** [review]

File: src/locales/en.json [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+  "active": "Active",
+  "import": "Import",
+  "lists": "Lists",
+  "custom": "Custom",
+  "copied": "Copied!",
+  "manage": "Manage",
+  "error": "Error",
+  "close": "Close",
+  "dismiss": "Dismiss"
+}
```
Comment (CedrikNikita) [inline src/locales/en.json:186]:
```suggestion
}

```

File: src/locales/fr.json [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+  "active": "Active",
+  "import": "Importer",
+  "lists": "Listes",
+  "custom": "Personnalisé",
+  "copied": "Copié!",
+  "manage": "Faire en sorte",
+  "error": "Erreur",
+  "close": "Proche",
+  "dismiss": "Rejeter"
+}
```
Comment (CedrikNikita) [inline src/locales/fr.json:186]:
```suggestion
}

```

File: src/components/TokenSelector/SelectTokenModal.vue [hunk 3]
```diff
@@ -171,12 +171,12 @@ export default {
     },
     modalTitle() {
       if (this.tab === 'manage-token-list') {
-        return 'Manage tokens';
+        return `${this.$t('manage')}`;
```
Comment (CedrikNikita) [inline src/components/TokenSelector/SelectTokenModal.vue:174]:
```suggestion
        return this.$t('manage');
```

**Review by CedrikNikita — COMMENTED** [review]
What else is missing translations:

`PoolView`: 
- `Add liquidity`

`AddLiquidity` view: 
- `Supply`, `Approve`
- `generateAddLiquidityMessage`

`SwapView`:
- `generateSwapMessage`

`aetrenity.js`:
- `addTransaction` info in `createPairAllowance` and in 'createTokenAllowance'

**Comment by kenodressel** [comment]
Id still merge this now and move the missing stuff to an issue.

**Comment by CedrikNikita** [comment]
> Id still merge this now and move the missing stuff to an issue.

Every other raised problems can be fixed fast.

File: src/components/MainWrapper.vue [hunk 4]
```diff
@@ -157,7 +155,7 @@ export default {
     },
     back() {
       if (!this.$store.state.route.from.name) {
-        this.$router.push({ name: 'swap' });
+        this.$router.push({ path: '/swap' });
```
Comment (CedrikNikita) [inline src/components/MainWrapper.vue:158]: I don't think that this change should be a part of this PR.

File: src/components/Tip.vue [hunk 5]
```diff
@@ -1,7 +1,7 @@
 <!-- eslint-disable vue/multi-word-component-names -->
 <template>
   <div class="tip">
-    {{ ` ${tip}` }}
+    <b>{{ $t('tip') }}:</b>{{ ` ${tip}` }}
```
Comment (CedrikNikita) [inline src/components/Tip.vue:4]: Arent we agreed not to show `Tip:` anymore?

**Comment by CedrikNikita** [comment]
To not being too annoying i've approved this PR, but found something that in my opinion would be better to fix before merging.

**Pull request merged**